websites/M/MLPOL/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "StrangeUniverseArtist",
+    "id": "727029933255163946"
+  },
+  "url": "mlpol.net",
+  "description": {
+    "en": "Welcome to the Mongolian political horsewhispering forum, also known as /mlpol/"
+  },
+  "service": "mlpol",
```

Comment:
service name and folder name must be the same (caps inc.)

Reply:
Done